input/node-prizes.txt

```diff
@@ -1,3 +1,3 @@
-NODEID	prize	active
+NODEID	prize	active	dummy
```

Comment:
I suggest modifying the example data file `node-prizes.txt` to add a dummy column there as well.

Reply:
should i add `A` or `C` to be the `dummy` node?

Reply:
TODO: a is dummy node

Reply:
We can use A to match the sources